emily/handler/src/api/models/deposit/mod.rs

```diff
@@ -79,7 +79,7 @@ pub struct DepositParameters {
     /// the transaction.
     pub max_fee: u64,
     /// Bitcoin block height at which the reclaim script becomes executable.
-    pub lock_time: u64,
+    pub lock_time: i64,
```

Comment:
Why `i64` now? Can this be negative?

Reply:
We can actually make this a `u64` if we add a getter to the `ReclaimScriptInputs` struct for the `lock_time` that returns a `u64`. When you create that struct we check that the `lock_time` is positive, so the `TryFrom` conversion will always succeed. Hmmm, there are a few small fixes that I'm going to be pushing, I'll add this in there.

Reply:
I added a getter in the reclaim script type in https://github.com/stacks-network/sbtc/pull/642 (along with some other slight changes)